src/main/java/com/redhat/devtools/lsp4ij/usages/LSPUsageSearcher.java

```diff
@@ -53,11 +55,30 @@ public class LSPUsageSearcher extends CustomUsageSearcher {
 
     @Override
     public void processElementUsages(@NotNull PsiElement element, @NotNull Processor<? super Usage> processor, @NotNull FindUsagesOptions options) {
+
+        PsiFile file = element.getContainingFile();
+        if (file == null) {
+            return;
+        }
+
+        VirtualFile virtualFile = file.getVirtualFile();
+        if (virtualFile == null) {
+            return;
+        }
+
+        Project project = element.getProject();

```

Comment:
Don't proceed unless there's at least one configured language server definition for this file that supports usages. That really addresses Joachim's core issue where it was doing this work in a file that shouldn't have been considered by LSP4IJ at all.

Reply:
Great!